README.md

```diff
@@ -55,5 +55,5 @@ HAPROXY_TIMEOUT_CHECK=100ms  # haproxy's 'timeout check' option
 
 DISCOVERY_METHOD=dns  # currently required, but this is the only option. This is to future proof.
 DISCOVERY_DNS_REFRESH=5s  # how often to query DNS record
-DISCOVERY_DNS_NAME=  # The SRV DNS record to use for peers. This must be the form of `_[service]._[proto]`
+DISCOVERY_DNS_NAME=  # The SRV DNS record to use for peers. This must be the form of `_[service]._[proto]._[domain].`
```

Comment:
Why would it be _[domain]? This isn’t right.

Reply:
Do you prefer `_[name]`?
I'm just looking at what we have for symbolicator (`DISCOVERY_DNS_NAME=_peers._tcp.symbolicator-peers.default.svc.cluster.local.`) and it looks like there's the third component missing in the comment.

Reply:
I meant specifically to highlight the underscore prefixes. Since this is a specific format for SRV records.

So let’s go with `_service._proto.name` to match documentation here. https://golang.org/pkg/net/#LookupSRV

Reply:
Also how it’s called out in the RFC. https://tools.ietf.org/html/rfc2782

Reply:
LGTM, and yeah, dunno why I put that underscore there, fixing.